### PR TITLE
feat(clerk-js): Sign In Retheme

### PR DESCRIPTION
**Pull request by octoper**
## Checklist

- [ ] `npm test` runs as expected.
- [ ] `npm run build` runs as expected.
- [ ] (If applicable) [JSDoc comments](https://jsdoc.app/about-getting-started.html) have been added or updated for any package exports
- [ ] (If applicable) [Documentation](https://github.com/clerk/clerk-docs) has been updated

## Type of change

- [ ] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

## Packages affected

- [ ] `@clerk/backend`
- [ ] `@clerk/chrome-extension`
- [X] `@clerk/clerk-js`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/fastify`
- [ ] `gatsby-plugin-clerk`
- [X] `@clerk/localizations`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/remix`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/shared`
- [ ] `@clerk/themes`
- [ ] `@clerk/types`
- [ ] `build/tooling/chore`

### CODE REVIEW

**Comment by octoper** [comment]
!preview

**Comment by clerk-cookie** [comment]
Hey @octoper, your preview is currently deploying and should be available shortly.

| Status | Preview | Updated (UTC) |
| :-- | :-- | :-- |
| :cook: _Deploying..._ | [Inspect](https://javascript-preview-qq2q2vtb0.clerkstage.dev) | Dec 15, 2023 03:31 PM |

File: packages/clerk-js/src/ui/elements/IdentityPreview.tsx [hunk 1]
```diff
@@ -30,12 +30,7 @@ export const IdentityPreview = (props: IdentityPreviewProps) => {
   );
 
   if (!refs.current.identifier) {
-    return (
-      <Container {...rest}>
-        <Authenticator />
```
Comment (desiprisg) [inline packages/clerk-js/src/ui/elements/IdentityPreview.tsx:35]: why are we removing this

Comment (desiprisg) [inline packages/clerk-js/src/ui/elements/IdentityPreview.tsx:35]: Why are we removing this?

Comment (octoper) [reply packages/clerk-js/src/ui/elements/IdentityPreview.tsx:35]: I was following the designs and that part has been removed.

Comment (desiprisg) [reply packages/clerk-js/src/ui/elements/IdentityPreview.tsx:35]: Is that just the icon ? Can you still sign in w./ auth app normally?

Comment (octoper) [reply packages/clerk-js/src/ui/elements/IdentityPreview.tsx:35]: yeah it's only the icon and the text, everything else works normally!

**Review by anagstef — APPROVED** [review]

**Pull request merged**